torch/lib/TH/generalFunc.h

```diff
@@ -0,0 +1,18 @@
+#ifndef GENERAL_FUNC_H
+#define GENERAL_FUNC_H
+
+ptrdiff_t SearchingIndex(ptrdiff_t index, long *stride, int dim, long* size)
```

Comment:
With the Windows PRs, we have replaced `long` by `int64_t`. Could you do those changes here as well?

Reply:
I will replace these with the lateset data type.